.docker/Dockerfile

```diff
@@ -192,7 +192,26 @@ RUN . "/opt/ros/${ROS_DISTRO}/setup.sh" \
     && colcon build
 
 # Setup the simulation environment variables
-RUN echo "source ${USER_WORKSPACE}/src/blue/.docker/entrypoints/sim.sh" >> /home/$USERNAME/.bashrc
+RUN  <<EOT cat >> /home/$USERNAME/.bashrc
+
+# Add results of ArduSub build
+export PATH=\$HOME/ardupilot/build/sitl/bin:\$PATH
+
+# Optional: add autotest to the PATH, helpful for running sim_vehicle.py
+export PATH=\$HOME/ardupilot/Tools/autotest:\$PATH
+
+# Add ardupilot_gazebo plugin
+export GZ_SIM_SYSTEM_PLUGIN_PATH=\$HOME/ardupilot_gazebo/build:\$GZ_SIM_SYSTEM_PLUGIN_PATH
+
+# Add ardupilot_gazebo models and worlds
+export GZ_SIM_RESOURCE_PATH=\$HOME/ardupilot_gazebo/models:\$HOME/ardupilot_gazebo/worlds:\$GZ_SIM_RESOURCE_PATH
+
+# Add blue models and worlds
+export GZ_SIM_RESOURCE_PATH=\$HOME/ws_blue/src/blue/blue_description/gazebo/models:\$HOME/ws_blue/src/blue/blue_description/gazebo/worlds:\$GZ_SIM_RESOURCE_PATH
+
+# Add blue meshes
+export GZ_SIM_RESOURCE_PATH=\$HOME/ws_blue/src/blue/blue_description/meshes:\$GZ_SIM_RESOURCE_PATH
```

Comment:
Does `${USER_WORKSPACE}` work here? If so, we can shorten this a bit using that variable.

Reply:
Added an intermediate variable (in shell, not in Docker) `$BLUE_DESCRIPTION` which points to `.../blue_description/`